<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Student</class>
 <widget class="QMainWindow" name="Student">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>693</width>
    <height>541</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <property name="spacing">
     <number>5</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>ËØæÁ®ãË°®</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="leftMargin">
         <number>15</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>15</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,13">
          <item>
           <widget class="QComboBox" name="comboBox">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>0</height>
             </size>
            </property>
            <item>
             <property name="text">
              <string>ÂÖ®ÈÉ®</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>‰ªäÂ§©</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Êú¨Âë®</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Êú¨Êúà</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableView" name="tableView">
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>‰∏™‰∫∫ËµÑÊñô</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,2,2,1,8">
          <property name="spacing">
           <number>13</number>
          </property>
          <property name="leftMargin">
           <number>20</number>
          </property>
          <property name="topMargin">
           <number>20</number>
          </property>
          <property name="rightMargin">
           <number>20</number>
          </property>
          <property name="bottomMargin">
           <number>20</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="cancel">
              <property name="text">
               <string>ÂèñÊ∂à</string>
              </property>
              <property name="shortcut">
               <string>Ctrl+Z</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="save">
              <property name="text">
               <string>‰øùÂ≠ò</string>
              </property>
              <property name="shortcut">
               <string>Ctrl+S</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,5,1,4,1,2">
            <item>
             <widget class="QLabel" name="label">
              <property name="font">
               <font>
                <pointsize>16</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Âßì    Âêç</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="le_name"/>
            </item>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="font">
               <font>
                <pointsize>16</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Âπ¥        ÈæÑ</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="le_age"/>
            </item>
            <item>
             <widget class="QLabel" name="label_3">
              <property name="font">
               <font>
                <pointsize>16</pointsize>
               </font>
              </property>
              <property name="text">
               <string>ÊÄß  Âà´</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="sex">
              <item>
               <property name="text">
                <string>Áî∑</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Â•≥</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="4,17,4,27">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="font">
               <font>
                <pointsize>16</pointsize>
               </font>
              </property>
              <property name="text">
               <string>ÊâãÊú∫Âè∑</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="le_phone"/>
            </item>
            <item>
             <widget class="QLabel" name="label_5">
              <property name="font">
               <font>
                <pointsize>16</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Ë∫´‰ªΩËØÅÂè∑</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="le_id"/>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox">
            <property name="font">
             <font>
              <pointsize>26</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">            border-color: rgb(156, 156, 156);
            border-width: 1px;
            border-style: solid;
            margin-top: 0.5ex;</string>
            </property>
            <property name="title">
             <string>üìÜ</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <item>
                <widget class="QLabel" name="label_7">
                 <property name="font">
                  <font>
                   <pointsize>16</pointsize>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true"> border-width: 0px;</string>
                 </property>
                 <property name="text">
                  <string>Êú¨ÊúàËØæÁ®ãÊï∞Ôºö</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="num1">
                 <property name="font">
                  <font>
                   <pointsize>16</pointsize>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true"> border-width: 0px;</string>
                 </property>
                 <property name="text">
                  <string>----</string>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_7"/>
               </item>
               <item>
                <widget class="QLabel" name="label_8">
                 <property name="font">
                  <font>
                   <pointsize>16</pointsize>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true"> border-width: 0px;</string>
                 </property>
                 <property name="text">
                  <string>Ââ©‰ΩôËØæÁ®ãÊï∞Ôºö</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="num2">
                 <property name="font">
                  <font>
                   <pointsize>16</pointsize>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true"> border-width: 0px;</string>
                 </property>
                 <property name="text">
                  <string>----</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>693</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Êï∞ÊçÆÂ∫ì</string>
    </property>
    <addaction name="gohome"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="gohome">
   <property name="text">
    <string>üèö</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
